data/mapcss-mapping.csv

```diff
@@ -1720,3 +1720,13 @@ man_made|water_tap|drinking_water_no;[man_made=water_tap][drinking_water=not],[m
 man_made|water_well|drinking_water_no;[man_made=water_well][drinking_water=not],[man_made=water_well][drinking_water=no],[man_made=water_well][drinking_water:legal=no];;;;1605;
 natural|spring|drinking_water_no;[natural=spring][drinking_water=not],[natural=spring][drinking_water=no];;;;1606;
 amenity|water_point|drinking_water_no;[amenity=water_point][drinking_water=not],[amenity=water_point][drinking_water=no];;;;1607;
+shop|pasta;1608;
+attraction|water_slide;1609;
```

Comment:
actually almost all (20k out of 21k) of water_slides are WAYS, not nodes...
does it make sense to try display it with an icon? (will it work at all?)

Reply:
No, an icon doesn't make sense. Many _water_sliede_ ways consist of several sections tagged with different _level_, so icon rendering would display many duplicates.

(Icons on ways don't work, at least they don't work for _bus_stop_)

Reply:
You are right, will remove node style. Any ideas on line rules?

Reply:
same for `roller_coaster` and `maze`..

Reply:
Lines could be similar to highway=raceway, but a tad thinner and different color. Maybe brownish (like "tourism" POIs) or maybe light violet (not very bright).

Reply:
I checked a few places on overpass turbo: `roller_coaster` and `maze` are not a problem because they are used on areas / closed lines. 

(Side node: There is the separate `roller_coaster=track` tag for the tracks of the roller coaster)

Reply:
are you going to add a line style for water_slide in a next PR?
as its not used RN